infra/ballista/build/lambda-handler.py

```diff
@@ -0,0 +1,105 @@
+import base64
+import os
+import subprocess
+import logging
+import sys
+import shutil
+import time
+
+
+logging.getLogger().setLevel(logging.DEBUG)
+__stdout__ = sys.stdout
+process_s = 0
+process_e = 0
```

Comment:
It is misleading to initialize these as integers if they will become process objects later on (None would likely be better)

Reply:
I like to use the non typing capacity of python to the max. but if you think this might be clearer for the reader I can change it